Northstar.CustomServers/mod/scripts/vscripts/mp/_classic_mp_dropship_intro.gnut

```diff
@@ -255,4 +256,4 @@ void function PlayerJumpsFromDropship( entity player )
 		WaitFrame()
 	
 	TryGameModeAnnouncement( player )
-}
\ No newline at end of file
+}
```

Comment:
Unnecessary whitespace change, otherwise looks good

Reply:
IMO this whitespace change is fine, files should have an ending newline rly